### DISCUSSION TITLE
Creating issues and discussions in english

### DISCUSSION THREAD

**Discussion by eugenov** [kingToolbox/WindTerm#459] [General]
Dear maintainer, I have a proposition: could you please do the same at your repo?
https://github.com/Eugeny/tabby/issues/5532
Thank you for your hard work!

**Comment by kingToolbox** [comment]
Thank you very much, @eugenov, I totally agree with you, so let's just do it!

**Reply by eugenov** [reply]
What will be the policy? No eng - > deleting?

**Reply by kingToolbox** [reply]
I will translate for them, at least the title.:smile:

I have started doing this today. According to the broken window theory, when there are no posts in other languages, it will not take long for everyone to get used to posting in English.

**Reply by Kristories** [reply]
@kingToolbox Please see my [suggestion](https://github.com/Eugeny/tabby/issues/5631). Maybe this can help.

**Comment by liuqi3312** [comment]
How does the source code compile into an application

**Comment by slowwalker79** [comment]
It is recommended that the session log save the configuration items with global effect, which is too troublesome to save for each session, and sometimes it will be forgotten, and it cannot be verified later.

**Reply by slowwalker79** [reply]
Strongly recommended

**Comment by holyvan** [comment]
linux,release 2.5 and 2.4,     "terminal.text.autoCopySelection" : true, but it is not effective

**Comment by yjx1804** [comment]
锁屏密码在哪里清除？？？

**Comment by FloatLifeStudio** [comment]
I think it would be cooler to add Wake On LAN feature.

**Comment by tixo** [comment]
Disagree
不同意

**Comment by slowwalker79** [comment]
？？？
&nbsp;


slow&nbsp;walker
***@***.***



&nbsp;




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年7月18日(星期二) 下午4:56
收件人: ***@***.***&gt;; 
抄送: "slow ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [kingToolbox/WindTerm] Creating issues and discussions in english (Discussion #459)





 
Disagree
 不同意

**Comment by William3365** [comment]
When can add the one click import/export session feature?
